src/EditorFeatures/CSharpTest/Diagnostics/NamingStyles/NamingStylesTests_OptionSets.cs

```diff
@@ -66,6 +72,66 @@ private NamingStylePreferences ClassNamesArePascalCaseOption()
             return info;
         }
 
+        private NamingStylePreferences FieldNamesAreCamelCaseOption()
+        {
+            var symbolSpecification = new SymbolSpecification(
+                null,
+                "Name",
+                ImmutableArray.Create(new SymbolSpecification.SymbolKindOrTypeKind(SymbolKind.Field)),
+                ImmutableArray<Accessibility>.Empty,
+                ImmutableArray<SymbolSpecification.ModifierKind>.Empty);
+
+            var namingStyle = new NamingStyle(
+                Guid.NewGuid(),
+                capitalizationScheme: Capitalization.CamelCase,
+                name: "Name",
+                prefix: "",
+                suffix: "",
+                wordSeparator: "");
+            var namingRule = new SerializableNamingRule()
```

Comment:
nit: either add blank lines between all of the statements (preferred) or remove the one above.

Reply:
➡️ Fixed in 26fac2b0002c55f852b44438d1f9f79b84ee1b60